.github/workflows/shared_deploy_exchange.yml

```
@@ -28,6 +28,13 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      # Remove when switching to the new Exchange - BUSINESS_GROUP_ID_GLOBAL_2 completely for all ENV (DEV, STAGING, PROD)
+      - name: Modify groupId in pom.xml
+        shell: bash
+        run: |
+          perl -0777 -pe 's/<groupId>b1493a71-3550-4d96-a623-e3f11d038f62<\/groupId>/<groupId>${{ secrets.BUSINESS_GROUP_ID }}<\/groupId>/' -i.bak pom.xml
```

@andyduong1920 Could we just replace any string in the `groupId` with the value from ${{ secrets.BUSINESS_GROUP_ID }} instead of just the fixed value `b1493a71-3550-4d96-a623-e3f11d038f62`?
```suggestion
          perl -0777 -pe 's/<groupId>[^<]*<\/groupId>/<groupId>${{ secrets.BUSINESS_GROUP_ID }}<\/groupId>/' -i.bak pom.xml
```

---

Oh, you need that on the CBTL service, is it @hoangmirs Let's me check that

---

I have fixed that here @hoangmirs - thanks for the catch